updatehub/Cargo.toml

```diff
@@ -10,37 +10,37 @@ license = "Apache-2.0"
 edition = "2018"
 
 [dependencies]
-actix = "0.10.0-alpha.2"
+actix = { version = "0.10.0-alpha.2", default-features = false }
 actix-rt = "1"
-actix-web = "2"
+actix-web = { version = "3.0.0-alpha.1", default-features = false }
 argh = "0.1.3"
 async-trait = "0.1"
-chrono = { version = "0.4", features = ["serde"] }
+chrono = { version = "0.4", default-features = false, features = ["serde"] }
 compress-tools = { git = "https://github.com/OSSystems/compress-tools-rs.git" }
 derive_more = { version = "0.99", default-features = false, features = ["deref", "deref_mut"] }
 easy_process = "0.1"
 find-binary-version = "0.2"
 infer = "0.1"
 lazy_static = "1"
-ms-converter = "0.5"
+ms-converter = "0.7"
 nix = "0.17"
 openssl = "0.10"
 pkg-schema = { path = "../updatehub-package-schema", package = "updatehub-package-schema" }
 quale = "1"
-regex = "1"
-reqwest = { version = "0.10", features = ["default-tls", "json"] }
```

Comment:
We can't remove tls support as we need to send requests to `https` domains. And from the two alternatives reqwest offers, I belive `default-tls` will generate less dependencies for us.

Reply:
Indeed. I reverted this part and resent the patches.